packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,471 @@
+        left: var(--aa-spacing-half);
+        padding-top: var(--aa-spacing-half);
+        .aa-Item {
+            padding: 0;
+        }
+        .aa-SourceHeader {
+            margin: var(--aa-spacing-half) 0 var(--aa-spacing-half) 2px;
+        }
+    }
+    // when a query isn't resolved yet
+    &.aa-Panel--stalled {
+        filter: grayscale(1);
+        opacity: 0.5;
+        transition: opacity 200ms ease-in;
+    }
 }
 
-.aa-InputWrapper,
-.aa-TouchSearchButton {
-  align-items: center;
-  background-color: #fff;
-  display: flex;
-  height: 2.5rem;
-  position: relative;
-  width: 100%;
+// ----------------
+// Sources
+// children of .col-results
+// Each section can be styled independantly here
+// ----------------
+section.aa-Source {
+    width: 100%;
+    // source title
+    .aa-SourceHeader {
+        position: relative;
+        line-height: var(--aa-spacing);
+        margin: var(--aa-spacing-half) var(--aa-spacing);
+        // title typography
+        span {
+            color: var(--aa-primary-color);
+            display: inline-block;
+            position: relative;
+            z-index: 2;
+            padding-right: var(--aa-spacing-half);
+            font-size: 0.8em;
+            font-weight: 600;
+            text-transform: capitalize;
+            background: var(--aa-background-color);
+        }
+        // Line separator
+        .aa-SourceHeaderLine {
+            z-index: 1;
+            position: absolute;
+            display: block;
+            top: var(--aa-spacing-half);
+            left: 0;
+            right: 0;
+            height: 2px;
+            opacity: 0.3;
+            border-bottom: solid 1px var(--aa-primary-color);
+        }
+        // hide empty header
+        &:empty {
+            display: none;
+        }
+    }
+    &:empty {
+        // hide empty section
+        display: none;
+    }
+    // the list of results inside the source
+    ul.aa-List {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
 }
 
-.aa-Input,
-.aa-TouchSearchButton {
-  border: 1px solid #d6d6e7;
-  border-radius: 3px;
-  box-shadow: rgb(119 122 175 / 30%) 0 1px 4px 0 inset;
-  font: inherit;
+// ----------------
+// Hit Layout - smart layout
+// ----------------
+li.aa-Item {
+    display: flex;
+    padding: 0 var(--aa-spacing-half);
+    line-height: 1.1em;
+    align-items: center;
+    color: var(--aa-text-color);
+    height: 100%;
+    cursor: pointer;
+    // when the result is active
+    &[aria-selected='true'] {
+        background-color: var(--aa-selected-color);
+        .aa-ItemActionButton {
+            visibility: visible;
+        }
+    }
+    // wrap hit with url but we don't need to see it
+    a.aa-ItemLink {
+        color: inherit;
+        text-decoration: none;
+        width: 100%;
+        display: flex;
+        align-items: center;
+    }
+    // the result type icon inlined svg or img
+    .aa-ItemSourceIcon {
```

Comment:
I think that `.aa-ItemIcon` is enough. The word "source" here doesn't provide too much information.